docs/document/docs/guides/table-readonly-table-customization.md

```diff
@@ -0,0 +1,514 @@
+4. Table: Update cell value depending on the other cell value change
+5. Table: Table in table
+
+## Components to use
+- [Dropdown](../components/desktop/dropdown.md)
+- [Table](../components/desktop/table.md)
+- [Text](../components/desktop/text.md)
+- [ReadOnlyTable](../components/desktop/readonly-table.md)
+- [RadioButton](../components/desktop/radio-button.md)
+
+## Table and ReadOnlyTable usage difference
+Basically, the Table and ReadOnlyTable component have the same structure.
+
+The ReadOnlyTable component allows the user to display a read-only mode table. This means that it only displays plain text that the user cannot manipulate.</br>
+
+In contrast, the Table allows the user to manipulate the components displayed in the table cell such as changing data, adding and deleting records.
+
+The biggest difference between Table and ReadOnlyTable components is the columns property of the table have a render property (`columns.render`) to specify which elements to display in the cell that the user can interact with. In addition, columns also have `actionButton` property (`table.actionButton`) to show/hide the add/remove row button.
+
+## Understanding `change` event
+It will be easier for you to customize the table when you understand the `change` event flow in the Table.
+>The table cell will listen `change` event from the component inside the cell and assign that new value to the field.</br>
+
+### Create an App
+Create an app that includes a blank space field with the id "space".
+
+### JavaScript and CSS Customization
+When you import the UMD file of Kintone UI Component to the app, you can upload the JavaScript files by following these steps:<br>
+You can see how to upload a file in the [Quick Start](../getting-started/quick-start.md).
+
+
+### Prepare basic table
+Suppose we have the following settings to display username with Text component:
+```javascript
+const renderName = (cellData) => {
+  const text = new Kuc.Text({ value: cellData });
+  return text;
+};
+
+const columns = [{ title: "Username", field: "username", render: renderName }];
+const data = [{ username: "user1" }, { username: "user2" }];
+
+const table = new Kuc.Table({columns, data});
+const space = kintone.app.record.getSpaceElement('space');
+space.appendChild(table);
+```
+
+### The display on UI
+
+![render](assets/table-edit-text.gif)
+When you edit the input in the first cell, the Text component will trigger `change` event with `detail.value = "user1 edited"`.
+The table then catch that value and assigns it back to the `username` field.
+Now the data of the table will be:
+```javascript
+[{ username: "user1 edited" }, { username: "user2" }];
+```
+
+### Modify the event detail
+If you want to customize the value the table receives then you need to modify it in the `change` event before the table receives it.
+We also use the sample code above and add the following content to `renderName` function:
+
+```javascript
+// table.js
```

Comment:
Do we need this line??
```suggestion
```

Reply:
I think deleting it is ok. I just commented for the filename